src/styles/custom-components.less

```diff
@@ -192,6 +192,18 @@
         font-variant-ligatures: none;
     }
 
+    &[data-placeholder]::before {
+        color: var(--black-200);
+        position: absolute;
+        content: attr(data-placeholder);
+        pointer-events: none;
+    }
+    &.js-markdown[data-placeholder]::before {
```

Comment:
Don't add styling to `js-*` classes. If you want to target the markdown editor specifically, we can add a base class to that editor to accommodate.

Reply:
I removed the JS that sets this `.js-markdown` class and added `.md-editor` as a base class on the commonmark editor in https://github.com/StackExchange/Stacks-Editor/pull/116/commits/8d27d576305e8a193a5733fcc39f25f7e653c2ef. `.md-editor` seems more descriptive/appropriate but I'm still not 100% on it, so suggestions are welcome.